tests/codebase/test_js_license_set.py

```diff
@@ -45,5 +45,5 @@ def test_js_license_set() -> None:
     '''
     os.chdir('bokehjs')
     cmd = ["npx", "license-checker", "--production", "--summary", "--onlyAllow", ";".join(LICENSES)]
-    proc = run(cmd)
+    proc = run(cmd, shell=True)
```

Comment:
This will likely break other platforms. We've already iterated over this change before, when dealing with npm's regressions. What version of nodejs/npm do you have installed?

Reply:
@mattpap On Windows I have node.js v16.5.0, npm 7.20.0 ; macOS node.js v16.4.1, npm 7.20.0

Reply:
Do you have `npx.bat` on path? If so, I think, the best solution would be to use `npx.bat` on Windows and `npx` elsewhere, with no shell.

Reply:
Can you elaborate on the potential issue with shell? It "works" on my mac, but I guess I don't understand the downside...

On Windows  `npx` is recognized on path with the bkdev virtual environment active, but not on my conda base or system. `npx.bat` doesn't work.
I'm wondering if the subprocess doesn't recognize the bkdev environment without shell?

### Testing npx on path
npx call works in bkdev
```
(bkdev) C:\Users\parki\Documents\GitHub\Bokeh\bokeh\bokehjs>npx

Entering npm script environment at location:
C:\Users\parki\Documents\GitHub\Bokeh\bokeh\bokehjs
Type 'exit' or ^D when finished
```
npx call doesn't work in base
```
(bkdev) C:\Users\parki\Documents\GitHub\Bokeh\bokeh\bokehjs>conda deactivate
(base) C:\Users\parki\Documents\GitHub\Bokeh\bokeh\bokehjs>npx
'npx' is not recognized as an internal or external command,
operable program or batch file.
```
npx call doesn't work in system
```
(base) C:\Users\parki\Documents\GitHub\Bokeh\bokeh\bokehjs>conda deactivate
C:\Users\parki\Documents\GitHub\Bokeh\bokeh\bokehjs>npx
'npx' is not recognized as an internal or external command,
operable program or batch file.
```
### Windows-specific hacky fix without shell
This works, but I don't like it...
```
import sys
npx_path = os.path.join(sys.exec_prefix, 'npx.cmd')
cmd = [npx_path, "license-checker", ....
proc = run(cmd)
```

Reply:
Current path forward is to skip the test on Windows, so I'm marking this conversation resolved.